docs/asyncapi.yaml

```diff
@@ -121,7 +121,7 @@ components:
     commsAddresses:
       type: array
       title: Accepts an array of strings
-      description: List of contact email addresses (also accepts a single email address)
+      description: List of contact email addresses (accepts a single email address)
       example: ["mrMegaFarmer@strawberryFortFarm.co.uk", "msMegaFarmer@bluberryCottageFarm.com"]
       minItems: 1
       maxItems: 10
```

Comment:
For `maxItems` it is currently set to 10 but based on Abi's comment saying we don't need to cover this right now - should we leave `maxItems` as is or remove it completely and just have `minItems` set for the time being?